src/tests/unittests/asn1.c

```diff
@@ -439,6 +439,124 @@ static void torture_asn1_decode_entry_bit_string_ni(void **state)
 	assert_memory_equal(bit_string + 1, result, resultlen/8);
 }
 
+static void torture_asn1_put_tag_short(void **state)
+{
+	unsigned int tag = 0xAC;
+	const u8 expected[] = {0xAC, 0x01, 0x02};
+	const u8 data[] = {0x02};
+	size_t data_len = 1;
+	u8 out[10];
+	size_t out_len = sizeof(out);
+	u8 *p = out;
+	int rv;
+
+	/* Without the out and out_len we are getting expected length */
+	rv = sc_asn1_put_tag(tag, data, data_len, NULL, 0, &p);
+	assert_int_equal(rv, sizeof(expected));
+	assert_ptr_equal(p, out);
+
+	/* Now we do the actual encoding */
+	rv = sc_asn1_put_tag(tag, data, data_len, out, out_len, &p);
+	assert_int_equal(rv, SC_SUCCESS);
+	assert_memory_equal(out, expected, sizeof(expected));
+	assert_ptr_equal(p, out + sizeof(expected));
+
+	/* Short buffer */
+	rv = sc_asn1_put_tag(tag, data, data_len, out, 2, &p);
+	assert_int_equal(rv, SC_ERROR_BUFFER_TOO_SMALL);
+}
+
+static void torture_asn1_put_tag_long_tag(void **state)
+{
```

Comment:
clang-format suggested the following change:


```
-static void torture_asn1_put_tag_long_tag(void **state)
-{
+static void torture_asn1_put_tag_long_tag(void **state) {
```

<!-- identifier: clang-format -->
<!-- File: src/tests/unittests/asn1.c -->